src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -5876,7 +5873,7 @@ void MusicXMLParserPass2::harmony(const QString& partId, Measure* measure, const
             size_t nstaves = _pass1.getPart(partId)->nstaves();
             QString strStaff = _e.readElementText();
             int staff = _pass1.getMusicXmlPart(partId).staffNumberToIndex(strStaff.toInt());
-            if (staff >= 0 && staff < nstaves) {
+            if (staff >= 0 && staff < int(nstaves)) {
```

Comment:
I believe we'd prefer the mor verbose C++ casting `static_cast<int>(nstaves)`

Reply:
The most cases (at least in this file) are using the simple cast.